src/util/Constants.js

```diff
@@ -354,15 +354,16 @@ exports.WSEvents = {
   RELATIONSHIP_REMOVE: 'RELATIONSHIP_REMOVE',
 };
 
-exports.MessageTypes = {
-  0: 'DEFAULT',
-  1: 'RECIPIENT_ADD',
-  2: 'RECIPIENT_REMOVE',
-  3: 'CALL',
-  4: 'CHANNEL_NAME_CHANGE',
-  5: 'CHANNEL_ICON_CHANGE',
-  6: 'PINS_ADD',
-};
+exports.MessageTypes = [
```

Comment:
My PR #1236 already does this, with the exception of adding `'GUILD_MEMBER_JOIN'`